README.md

```diff
@@ -1,90 +1,33 @@
-
-```
-npm install monday-ui-react-core
-```
+Welcome to the Vibe Design System repository! This monorepo contains various open-source packages maintained by Vibe ([monday.com](https://www.monday.com)).
 
-## Usage
+## Overview
 
-Components are imported from the library's root entry:
+Vibe Design System offers a collection of packages designed to streamline your development process and enhance the user experience. Below is a list of our main packages:
 
-```javascript
-import { Button } from "monday-ui-react-core";
-```
-
-In order to load all the relevant CSS tokens, you should import the `tokens` file at the root of your application file
-
-```javascript
-import "monday-ui-react-core/tokens";
-```
+1. [core](packages/core/README.md): [monday.com](https://www.monday.com) React components library - [Storybook](https://style.monday.com).
+2. [style](packages/style/README.md): [monday.com](https://www.monday.com) styling foundations library.
 
-_If your project (or it's Storybook) is importing files differently - read more [here](DEPRECATED_IMPORTS.md)._
+Each package comes with its own README providing detailed information about its usage, installation instructions, and getting started guide.
 
-### Font installation
+## Installation
 
-We don't import fonts ourselves, we work best with the following fonts -
-Poppins, Figtree and Roboto, we recommend adding the following `link` import to your application
+Install the component library
 
-```html
-<link
-  href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-  rel="stylesheet"
-/>
 ```
-
-### Peer dependencies
-
-We are reliant on React and React DOM, we are using them as externals, and we don't package them to the package, so you must have them in your project
-
-## Storybook
-
-We are using storybook in order to develop the components independently of any consumer.
-run this to build & run the storybook locally:
-
-```bash
-npm run storybook
+npm install monday-ui-react-core
 ```
 
-the storybook will be served on `http://localhost:7007`
-
-## Developing locally with your consumer application
+## Getting Started
 
-When developing locally, we are using a npm functionality called npm link, this allows us to
-work locally on our package and use it in a different project without publishing.
-This functionality basically overrides the npm mapping between package name to its repo, and points it to where the package is located locally.
+If you're new to Vibe Design System, we recommend starting with our main package:
 
-### Troubleshooting local development
+- [core](packages/core/README.md): This is the core of the Vibe Design System. Follow the installation guide and getting started instructions to integrate it into your project.
```

Comment:
I think it's super confusing to have both sections (Installation and Getting Started), Installation can be a sub section in Getting Started with a referral to additional setup in the core README